src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/TypePreinit.cs

```diff
@@ -2756,9 +2727,9 @@ public override void WriteFieldData(ref ObjectDataBuilder builder, NodeFactory f
         private sealed class ReadOnlySpanValue : BaseValueTypeValue, IInternalModelingOnlyValue
```

Comment:
Is ReadOnlySpanValue still the right name for this? If I am reading the code correctly, this can represent both readonly and non-readonly Span now.

Reply:
(The suggestion is to consider dropping `ReadOnly` from these names.)